usb/usb.go

```diff
@@ -59,6 +59,10 @@ rootwait
 		"uuid-dev",
 		"libssl-dev",
 	}
+	optional = map[string] bool {
```

Comment:
yes, I'm aware that the standard way is map[string]struct{}
But
this map is small
the code is clearer
so I want it this way.